pkg/postgres/migrations/migrator.go

```diff
@@ -115,9 +115,9 @@ func (m *Migrator) MigrateAll() error {
 		&_202411191550_operatorAVSSplits.Migration{},
 		&_202411191708_operatorPISplits.Migration{},
 		&_202411191947_cleanupUnusedTables.Migration{},
-		&_202412021311_stakerOperatorTables.Migration{},
 		&_202411221331_operatorAVSSplitSnapshots.Migration{},
 		&_202411221331_operatorPISplitSnapshots.Migration{},
+		&_202412021311_stakerOperatorTables.Migration{},
```

Comment:
This should actually go back to where it was on line 118

Reply:
The timestamp says it should come after though right?

Reply:
Done in [4075821](https://github.com/Layr-Labs/sidecar/pull/131/commits/4075821aa6aee768346bfa4b604486c8b934593f)